.github/workflows/ci.yml

```diff
@@ -301,17 +301,30 @@ jobs:
           workload_identity_provider: projects/345757944225/locations/global/workloadIdentityPools/github-actions-pool/providers/github-actions-provider
           service_account: github-actions-gcr@sentryio.iam.gserviceaccount.com
 
+      - name: 'Set up Cloud SDK'
+        uses: 'google-github-actions/setup-gcloud@v1'
+        with:
+          # https://github.com/google-github-actions/auth#authenticating-via-workload-identity-federation
+          # You must use the Cloud SDK version 390.0.0 or later to authenticate the bq and gsutil tools.
+          version: '>= 390.0.0'
+
       - name: Configure docker
         run: |
           gcloud auth configure-docker us.gcr.io
 
-      - name: Push to us.gcr.io
+      - name: Push to us.gcr.io and upload deployment assets
         run: |
           set -euxo pipefail
           docker pull -q "$IMG_VERSIONED"
           docker tag "$IMG_VERSIONED" "us.gcr.io/sentryio/relay:${{ github.event.pull_request.head.sha || github.sha }}"
           docker push "us.gcr.io/sentryio/relay:${{ github.event.pull_request.head.sha || github.sha }}"
 
+          docker run --rm --entrypoint cat "us.gcr.io/sentryio/relay:${{ github.event.pull_request.head.sha || github.sha }}" /opt/relay-debug.zip > relay-debug.zip
```

Comment:
Since we no longer need the symbols in Docker for the freight deployment, we should change the build to not even put the debug files in the docker container. Instead, we can just save them as a GitHub actions asset and then re-upload them to our devinfra.

Reply:
Can we punt this for now? Not sure how to approach that yet... it's built in the builder image and we'd need some way of copying it to the GHA runtime. Could upload the builder image but there's surely better ways.

Reply:
Yeah, fine with me 👍